crates/query-engine/sql/src/sql/ast.rs

```diff
@@ -225,10 +215,41 @@ pub enum BinaryArrayOperator {
 
 /// A scalar function
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub enum Function {
-    Coalesce,
-    JsonAgg,
-    Unknown(String),
+pub struct Function {
+    pub function_name: String,
+    pub is_infix: bool,
+}
+
+/// The Postgres 'coalesce' function for convenience.
```

Comment:
Cool. I actually think `helpers.rs` is a better place for these, this is where we have similar functions like `true_expr`, etc.